src/components/error-page/ErrorPage.jsx

```diff
@@ -4,22 +4,27 @@ import {
   Col,
   Container,
 } from '@edx/paragon';
+
 import SiteFooter from '@edx/frontend-component-footer';
 
 import ErrorPageHeader from './ErrorPageHeader';
-
+// Additional Notes, Consolidate button into children prop
 /**
  * React component for the error case when attempting to link a user to a customer. Renders
  * a header, error alert, and a footer.
+ * Can set params to display header and/or footer with bool value (true default)
+ * Redirect component can be added as a customized component (empty default)
  */
 const ErrorPage = ({
   title,
   subtitle,
+  showSiteHeader,
   showSiteFooter,
+  redirect,
   children,
 }) => (
   <>
-    <ErrorPageHeader />
+    {showSiteHeader && <ErrorPageHeader />}
```

Comment:
Oof, yeah... the current implementation of `ErrorPage` on `master` is less flexible than I originally thought... I had forgotten it includes the header and (optionally) footer. I thought it was more like the `NotFoundPage` component which it solely about the "Page not found" messaging, not anything to do with the page layout.

That said, an alternative approach to making `ErrorPage` more flexible to support our current use case might be to refactor `ErrorPage` to have sub-components that get composed together. That way, other pages could use the sub-components of `ErrorPage` as necessary in other pages, like the Executive Education (2U) page, for their styling.

Speaking of styling, it also turns out that the `ErrorPage` as currently implemented on `master` doesn't match the Figma spec (e.g., the color of the main heading should be our "danger" color) either 😅 We should probably fix that, too, while we're at it 😉 (in the below example, I'm using CSS utility classes instead of the existing custom CSS properties)?

```jsx
function ErrorPageTitle({ children }) {
  return (
    <h2 className="text-danger mb-5">{children}</h2>
  );
}

function ErrorPageSubtitle({ children }) {
  return (
    <h3 className="mb-4.5">{children}</h2>
  );
}

function ErrorPageContent({ children, className }) {
  return (
    <Container size="lg" className={classNames('error-page-container', className)}>
      {children}
    </Container>
  );
}

function ErrorPage({
  title,
  subtitle,
  // ...
}) {
  return (
    <ErrorPageHeader />
    <main id="content">
      <ErrorPageContent>
        <ErrorPageTitle>{title}</ErrorPageTitle>
        <ErrorPageSubtitle>{subtitle}</ErrorPageSubtitle>
        {children}
      </ErrorPageContent>
    </main>
  );
}
```

Then, to treat these as true sub-components when using them, we could do something like this in `ErrorPage.jsx`:

```jsx
function ErrorPage() { ... }

ErrorPage.Content = ErrorPageContent;
ErrorPage.Title = ErrorPageTitle;
ErrorPage.Subtitle = ErrorPageSubtitle;

export default ErrorPage;
```

such that in `ExecutiveEducation2UPage.jsx`, we could do something like, for example:

```jsx
function ExecutiveEducation2UError({ failureReason }) {
  // parse failure reason messaging based on `failureReason`

  return (
    <ErrorPageContent className="py-5">
        <Row>
          <Col xs={12} lg={{ span: 10, offset: 1 }}>
            <ErrorPageTitle>Error occurred while processing your request</ErrorPageTitle>
            <ErrorPageSubtitle>
              Your organization doesn&apos;t have sufficient balance to cover this course.
            </ErrorPageSubtitle>
            <p className="mb-6">
              Please contact your edX administrator or
              visit the edX help center to resolve the error and gain access to subsidized content, or
              continue to edX.org to start learning on your own.
            </p>
            {referrerURL && (
              <Button
                href={referrerURL}
                iconBefore={ArrowBack}
                variant="primary"
              >
                Return to previous page
              </Button>
            )}
          </Col>
        </Row>
      </ErrorPageContent>
  );
}

function ExecutiveEducation2UPage() {
  if (activeQueryParams.has('failure_reason')) {
    const failureReason = activeQueryParams.get('failure_reason');
    return (
      <ExecutiveEducation2UError failureReason={failureReason} />
    );
  }

  // ...
}
```